desktop-app/app/components/ScrollControls/index.js

```diff
@@ -55,6 +60,18 @@ const ScrollControls = ({
     width: 25,
   };
 
+  useEffect(() => {
```

Comment:
@durgakiran Why do we need this?

Reply:
To run whenever eventMirroring value changes and initial render check is to make sure toggle does not happen on  initial render